tasks/ca/statcan/geo.py

```diff
@@ -137,15 +133,17 @@ def input_files(self):
 
 class SimplifiedImportGeography(SimplifiedTempTableTask):
     resolution = Parameter(default=GEO_PR)
+    year = Parameter()
 
     def requires(self):
-        extension = 'gml' if self.resolution == GEO_FSA else 'shp'
-        return ImportGeography(resolution=self.resolution, extension=extension)
+        extension = 'gml' if self.resolution == GEO_FSA and self.year == '2011' else 'shp'
```

Comment:
There is a `GEOGRAPHY_FORMAT` constant that seems to be somewhat related to this. Could they be unified?